admin/services/data-access/completed-check.data.service.js

```diff
@@ -80,14 +154,37 @@ completedCheckDataService.findUnmarked = async function (batchSize) {
   return batchIds.map(b => b._id)
 }
 
+/**
+ * Return plain completedCheck objects that have not been marked, up to a the limit specified by `batchsize`
```

Comment:
I think the documentation and the code are out of step.  This returns an array of numbers (id's) not objects.